src/AppBundle/Tests/Controller/UserControllerTest.php

```diff
@@ -13,4 +13,14 @@ public function testGet()
 
         $this->assertJsonResponse($response, 200);
     }
+
+    public function testUserGet()
+    {
+        $client   = static::createClient();
+        $crawler  = $client->request('GET', '/users/54d1de7530d0c3bf178b4567');
```

Comment:
if it user id, so after you reload fixtures this id not exists anymore
use something like that:
```php
$userRepository->findOne([])->getId();
```